### DISCUSSION TITLE
When merged, are the images in the new file changed in anyway?

### DISCUSSION THREAD

**Discussion by Adventure4Life** [pdfarranger/pdfarranger#746] [Q&A]
I am freelancing a job at the moment that has a lot of stuff to print. The printers require me to supply a single PDF file with a CYMK image on each page and embeded colour profile. 

Now Photoshop can save out the PDFs correctly for each image... but I am curious as to if this application dose any modifications to the images during the merge or if it is "lossless". 

Also in the Photoshop saved PDFs have an option to save colour space profiles.. is this information also in the merged PDF file created?

**Comment by jeromerobert** [comment]
At the moment, PDF Arranger does not modify the images included in the imported PDF files. But I think that's not the question as color profiles are probably specified out side the image (i.e. in the document structure). According to the PDF specification, color spaces can be specified by objects (ex: images) or by pages. In that case it should work as expected. If the color space is specified at document level and you merge multiple document with different color profiles it will probably not work as expected.

I suppose any decent PDF viewer will give you this information.

**Comment by Adventure4Life** [comment]
Thank you for your reply. As you can probably tell I do not do much print work so really do not know much about the PDF format. You wouldn't happen to know is these options in the Photoshop save PDF dialogue is what you are talking about?

https://i.imgur.com/2UzjWRJ.jpg

Like if I set it to "colour conversion" that actually edits the colour information in the file, which will then work wit hyour merge app.. and I do not embed the colour information in the PDF itself.

Is this what you mean? (I know this is a little out of the scope of what this comment section is for, so if you do not know or do not have time to respond, I understand)

Thanks
--aNewHobby
